packages/template-retail-react-app/app/ssr.js

```diff
@@ -60,7 +60,7 @@ const options = {
     // environment variable as this endpoint will return HTTP 501 if it is not set
     useSLASPrivateClient: false,
     applySLASPrivateClientToEndpoints:
-        /oauth2\/(token|passwordless|password\/(login|token|reset|action))/,
+        /oauth2\/(token|authorize|passwordless\/(login|token)|password\/(reset|action))/,
```

Comment:
Now that we're supporting passwordless login, social login, etc, I think we should have this regex as the default. In other words, the value for `applySLASPrivateClientToEndpoints` should be in the pwa-kit-runtime and not here in the retail-react-app template. 

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/458f911ea072acd4d909c6504ddf16273e3e80db/packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js#L138

Having it in the runtime will be an easier change to consume by the customers. Otherwise, how would they know to update their existing project such that it configures the applySLASPrivateClientToEndpoints option?

Reply:
@vcua-mobify since you originally wrote the code for `applySLASPrivateClientToEndpoints`, what should we do in this case? ^

Reply:
I agree with @vmarta 

`applySLASPrivateClientToEndpoints` can be overridden by defining a `applySLASPrivateClientToEndpoints` inside the options object in ssr.js but since these will be a default supported endpoint, let's update the endpoint in the runtime